test/index.test.js

```diff
@@ -1,4 +1,4 @@
-var dateString = require('../lib/index.js');
+var dateString = require('../src/index.js');
```

Comment:
I feel like it makes more sense to test the src rather than the published?

Otherwise you would have to build it then test it each time?

Reply:
Probably a good idea to test both tbh. Incase something is changed during transpiled. When devving locally, use the src, when remote on Travel lib.